src/main/kotlin/blackjack/domain/Card.kt

```diff
@@ -0,0 +1,14 @@
+package blackjack.domain
+
+import blackjack.enums.CardShape
+import blackjack.enums.CardType
+
+class Card(
+    val shape: CardShape,
+    val type: CardType
+) {
+    val point
+        get() = type.point
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val point = type.point
```